### PR TITLE
138827713 add me resource api version5

### PR DESCRIPTION
**Pull request by lucyconklin**
This adds the me resource, and tests for valid attributes.

All tests passing.

### CODE REVIEW

File: Contributing.md [hunk 1]
```diff
@@ -10,15 +10,24 @@ To get your environment setup for contribution:
 cp .env.example .env
 ```
 
-2. Replace the values of the `.env` variables with your PivotalTracker credentials.
+2. Replace the variables in your `.env` file with your PivotalTracker credentials.
```
Comment (jasonnoble) [inline Contributing.md]: While I agree with these changes, they don't seem related to this story?

File: lib/pivotal-tracker/me.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class PivotalTracker::Me
```
Comment (jasonnoble) [inline lib/pivotal-tracker/me.rb]: Please inherit from our new `PivotalTracker::Resource` class (introduced as part of https://github.com/jasonnoble/pivotal-tracker/pull/11/files#diff-db66140f03dd10afa01447b572c04b2dR1)

File: lib/pivotal-tracker/me.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  attr_accessor :api_token, :created_at, :email, :has_google_identity, :id,
+                :initals, :kind, :name, :personal_settings, :project_ids,
+                :receives_in_app_notifications, :time_zone, :updated_at,
+                :username, :workspace_ids
+
+  def initialize(attributes)
```
Comment (jasonnoble) [inline lib/pivotal-tracker/me.rb]: Inheriting (see above) will lead to the removal of this method...

**Review by jasonnoble — CHANGES_REQUESTED** [review]
Needs rebasing and merge conflict resolution...

File: lib/pivotal-tracker/account.rb [hunk 4]
```diff
@@ -1,14 +1,10 @@
-class PivotalTracker::Account
+require './lib/pivotal-tracker/resource'
```
Comment (jasonnoble) [inline lib/pivotal-tracker/account.rb]: Is this required?

Comment (lucyconklin) [reply lib/pivotal-tracker/account.rb]: nope! Removing and running tests again now...
